src/platforms/php/guides/laravel/configuration/laravel-options.mdx

```diff
@@ -76,56 +77,48 @@ Learn more in [Closures and Config Caching](/platforms/php/guides/laravel/config
     'queue_job_transactions' => env('SENTRY_TRACE_QUEUE_ENABLED', false),
 
     // Capture queue jobs as spans when executed on the sync driver
-    'queue_jobs' => true,
+    'queue_jobs' => env('SENTRY_TRACE_QUEUE_JOBS_ENABLED', true),
 
     // Capture SQL queries as spans
-    'sql_queries' => true,
+    'sql_queries' => env('SENTRY_TRACE_SQL_QUERIES_ENABLED', true),
 
-    // Try to find out where the SQL query originated from and add it to the query spans
-    'sql_origin' => true,
+    // Capture where the SQL query originated from on the SQL query spans
+    'sql_origin' => env('SENTRY_TRACE_SQL_ORIGIN_ENABLED', true),
 
-    // Capture views as spans
-    'views' => true,
+    // Capture views rendered as spans
+    'views' => env('SENTRY_TRACE_VIEWS_ENABLED', true),
 
     // Capture Livewire components as spans
-    'livewire' => true,
+    'livewire' => env('SENTRY_TRACE_LIVEWIRE_ENABLED', true),
 
     // Capture HTTP client requests as spans
-    'http_client_requests' => true,
+    'http_client_requests' => env('SENTRY_TRACE_HTTP_CLIENT_REQUESTS_ENABLED', true),
 
     // Capture Redis operations as spans (this enables Redis events in Laravel)
     'redis_commands' => env('SENTRY_TRACE_REDIS_COMMANDS', false),
 
-    // Try to find out where the Redis command originated from and add it to the command spans
-    'redis_origin' => true,
-
-    // Indicates if the tracing integrations supplied by Sentry should be loaded
-    // See all default integration: https://github.com/getsentry/sentry-laravel/tree/master/src/Sentry/Laravel/Tracing/Integrations
-    'default_integrations' => true,
-
-    // Indicates that requests without a matching route should be traced
-    'missing_routes' => false,
-],
-```
+    // Capture where the Redis command originated from on the Redis command spans
+    'redis_origin' => env('SENTRY_TRACE_REDIS_ORIGIN_ENABLED', true),
 
-## Controller Base Namespace
+    // Enable tracing for requests without a matching route (404's)
+    'missing_routes' => env('SENTRY_TRACE_MISSING_ROUTES_ENABLED', false),
 
-Performance monitoring and router breadcrumbs can report the controller classname that handled the request.
+    // Configures if the performance trace should continue after the response has been sent to the user until the application terminates
+    // This is required to capture any spans that are created after the response has been sent like queue jobs dispatched using `dispatch(...)->afterResponse()` for example
+    'continue_after_response' => env('SENTRY_TRACE_CONTINUE_AFTER_RESPONSE', true),
 
-These names are generally longer than needed, that's why we strip away the base namespace that is the same for all controller classes.
-
-In Laravel this is `App\Http\Controllers` by default but you can override this using the `controllers_base_namespace` option:
-
-```php {filename:config/sentry.php}
-'controllers_base_namespace' => env('SENTRY_CONTROLLERS_BASE_NAMESPACE', 'App\\Http\\Controllers'),
```

Comment:
just want to double check that you meant to delete this info

Reply:
Yep, this was removed in the last major version of the SDK. I forgot to remove it back then.